src/installer.js

```diff
@@ -112,16 +113,55 @@ var getSize = function (options, callback) {
 }
 
 /**
+ * Determine the dependencies for the `shell.moveItemToTrash` API.
+ */
+var getTrashDepends = function (electronVersion) {
+  if (semver.lt(electronVersion, '1.4.1')) {
+    return 'gvfs-bin'
+  } else if (semver.lt(electronVersion, '1.7.2')) {
+    return 'kde-cli-tools | kde-runtime | trash-cli | gvfs-bin'
+  } else {
+    return 'kde-cli-tools | kde-runtime | trash-cli | libglib2.0-bin | gvfs-bin'
+  }
+}
+
+/**
+ * Determine what the default dependencies should be, based on the Electron
+ * version in use.
+ */
+var getDefaultDepends = function (options, callback) {
+  fs.readFile(path.resolve(options.src, 'version'), (err, tag) => {
+    if (err) return callback(err)
+
+    // The content of the version file is the tag name, e.g. "v1.8.1"
+    var version = tag.toString().slice(1).trim()
+
+    return callback(null, [
```

Comment:
Are we missing `libdbus-1-3`? it's a dependency on the [build script](https://github.com/electron/electron/blob/master/script/cibuild#L14)
Also, what about `libx11-6`? don't we always need this for anything GUI related? I guess `libxtst6` and `libgtk2.0-0` depends on it so it might not be needed to be explicitly added. On the other hand, many of these packages depend on `libc6`, and we're explicitly adding it.
`libasound2` might also be necessary and none of the current dependencies requires this library.

All these new suggestions are coming from the `atom` package that webupd8 builds on their own PPA. It has a complete set of dependencies than the default `.deb` from the official atom build.
You can see the dependencies needed on their [build log](https://launchpadlibrarian.net/321622385/buildlog_ubuntu-xenial-amd64.atom_1.17.2-1~webupd8~0_BUILDING.txt.gz) It's almost at the end, in a section called:
+------------------------------------------------------------------------------+
| Package contents
+------------------------------------------------------------------------------+